go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus-community/elasticsearch_exporter
 
-go 1.17
+go 1.19
```

Comment:
This one doesn't need to change for a go version bump although I think bumping it to at least 1.18 improves build dependency information. In this case I think it's fine unless @SuperQ has any objections.